scanomatic/ui_server_data/js/ccc/containers/PolynomialConstructionContainer.jsx

```diff
@@ -20,10 +20,8 @@ export default class PolynomialConstructionContainer extends React.Component {
     };
 
     this.handleConstruction = this.handleConstruction.bind(this);
-    this.handleConstructionResults = this.handleConstructionResults
-      .bind(this);
-    this.handleConstructionResultsError =
-            this.handleConstructionResultsError.bind(this);
+    this.handleConstructionResults = this.handleConstructionResults.bind(this);
```

Comment:
the functions ought not to be bound here but instead be arrow functions on the class, but that's probably better a separate issue.

Reply:
Sure, I don't understand the problem though, so I'd appreciate if you create it or at least commit to fleshing it out if I do.

Reply:
it's annoying that the way `this` works in those is redefined here in the constructor instead of them being defined as intended from the beginning. It has to do with javascript prototypes, so if interested google about the differences between function and arrow statement and how this affects `this`.

Reply:
I am not really that interested in anything except to know whether you will create the issue or if I should. :)

Reply:
i can make the issue